{{cookiecutter.repostory_name}}/nginx/templates/default.conf.template

```diff
@@ -132,6 +137,10 @@ server {
     access_log                /dev/stdout;
     error_log                 /dev/stderr info;
 
+    {% if cookiecutter.nginx_filter_bad_host_headers == 'y' -%}
+    include /etc/nginx/config_helpers/ignore_bad_host_allow_ip.conf;
+    {%- endif %}
```

Comment:
monitoring by us is typically done through autodiscovery from AWS or over service provider
I.e. we query by IPs/domains assigned to a particular ec2 machine, i.e. it is almost never $NGINX_HOST

so this this won't really help us.

Either it should allow all valid domains/adresses here OR we just silence Invalid HTTP_HOST header on sentry level.

Reply:
Nginx would be more performant, but it seems like resolving domains could be more complexity than it's worth. I changed to ignoring the logger on sentry level